site/docs/pages/transaction/transaction.mdx

```diff
@@ -123,6 +123,63 @@ export default function TransactionComponents() {
   </TransactionWrapper>
 </App>
 
+## `Transaction` with Paymaster Gas Sponsorship Capabilities
+##### Obtain a Paymaster and Bundler endpoint from the [Coinbase Developer Platform Paymaster](https://portal.cdp.coinbase.com/products/bundler-and-paymaster).
+
+<img
+  alt="OnchainKit Paymaster and Bundler endpoint"
+  title="OnchainKit Paymaster and Bundler endpoint"
+  src="/assets/onchainkit-components-paymaster-endpoint.png"
+  width="auto"
+/>
+```tsx
+// ommited for brevity
+<Transaction
+  address={address}
+  contracts={contracts}
+  capabilities={{ // [!code focus]
+    paymasterService: { // [!code focus]
+      url: process.env.PAYMASTER_AND_BUNDLER_ENDPOINT, // [!code focus]
+    }, // [!code focus]
+  }} // [!code focus]
+  >
+  <TransactionButton />
+  <TransactionSponsor text="OnchainKit" />
+</Transaction>
+```
+<App>
+  <TransactionWrapper>
+    {({ address, contracts, onError }) => {
+      if (address) {
+        return (
+          <Transaction 
+            address={address} 
+            contracts={contracts} 
+            onError={onError} 
+            capabilities={{
+              paymasterService: {
+                url: 'https://api.developer.coinbase.com/rpc/v1/base/KkgONfLFgUZKI8nnrttLbI78U2vyhtXg',
```

Comment:
Whos key are we using here? Should this be store in Vercel?

Reply:
This is my CDP key - Adding to Vercel